governance/votes/Executive vote - February 22, 2024.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Template - [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024
+summary: Stability Fee changes for ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C, Spark Protocol DC-IAM parameter changes, pushing USDP out of input conduit, yanking BA Labs DAI and MKR vest streams, triggering Spark proxy spell.  
+date: 2024-02-22T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Vote] Stability Fee Changes, Spark Protocol DC-IAM Parameter Changes, Housekeeping Actions, Spark Proxy Spell - February 22, 2024
+
+The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C [Stability Fees](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased, as detailed below. 
+- Spark Protocol [DC-IAM](https://manual.makerdao.com/module-index/module-dciam) parameters will be changed, as detailed below.
+- The following housekeeping actions will be carried out:
+  - BA Labs DAI and MKR vest streams will be yanked, as detailed below.
+  - USDP will be pushed out of the input conduit, as detailed below.
+- A Spark Protocol proxy spell will be triggered, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### ETH-A, ETH-B, ETH-C, WSTETH-A, WSTETH-B, WBTC-A, WBTC-B, WBTC-C [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Decreases
+
+**Authorization:** [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-9/23688/2)
```

Comment:
Please, add a backslash to generate a linebreak at the end of the authorization line.

```suggestion
**Authorization:** [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-9/23688/2)\
```